luda-editor/luda-editor-client/src/app/editor/mod.rs

```diff
@@ -1,18 +1,22 @@
 mod timeline;
 use self::{
-    clip_editor::{camera_clip_editor::image_browser::ImageBrowserItem, ClipEditor},
+    clip_editor::{
+        camera_clip_editor::image_browser::{ImageBrowserFile, ImageBrowserItem},
+        ClipEditor,
+    },
```

Comment:
```suggestion
    clip_editor::{camera_clip_editor::image_browser::ImageBrowserFile, ClipEditor},
```
ImageBrowserItem is not used anymore